cognite/client/_api/assets.py

```diff
@@ -213,9 +213,29 @@ def create(self, asset: Union[Asset, List[Asset]]) -> Union[Asset, AssetList]:
                 >>> res = c.assets.create(assets)
         """
         utils._auxiliary.assert_type(asset, "asset", [Asset, list])
-        if isinstance(asset, Asset) or len(asset) <= self._CREATE_LIMIT:
-            return self._create_multiple(asset)
-        return _AssetPoster(asset, client=self).post()
+        return self._create_multiple(asset)
+
+    def create_hierarchy(self, assets: List[Asset]) -> AssetList:
+        """Create asset hierarchy.
```

Comment:
You can add a little more information here about how this differs from .create(), i.e. which validation steps it will perform and how it will resolve the hierarchy locally and post assets in the correct order.